AutoOrganize/AutoOrganize.csproj

```diff
@@ -14,41 +14,41 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Remove="Web\autoorganizelog.html" />
-    <None Remove="Web\autoorganizelog.js" />
-    <None Remove="Web\autoorganizemovie.html" />
-    <None Remove="Web\autoorganizemovie.js" />
-    <None Remove="Web\autoorganizesmart.html" />
-    <None Remove="Web\autoorganizesmart.js" />
-    <None Remove="Web\autoorganizetv.html" />
-    <None Remove="Web\autoorganizetv.js" />
-    <None Remove="Web\fileorganizer.js" />
-    <None Remove="Web\fileorganizer.template.html" />
+    <None Remove="Web\autoorganizelog.html"/>
+    <None Remove="Web\autoorganizelog.js"/>
+    <None Remove="Web\autoorganizemovie.html"/>
+    <None Remove="Web\autoorganizemovie.js"/>
+    <None Remove="Web\autoorganizesmart.html"/>
+    <None Remove="Web\autoorganizesmart.js"/>
+    <None Remove="Web\autoorganizetv.html"/>
+    <None Remove="Web\autoorganizetv.js"/>
+    <None Remove="Web\fileorganizer.js"/>
+    <None Remove="Web\fileorganizer.template.html"/>
   </ItemGroup>
 
   <ItemGroup>
-    <EmbeddedResource Include="Web\autoorganizelog.html" />
-    <EmbeddedResource Include="Web\autoorganizelog.js" />
-    <EmbeddedResource Include="Web\autoorganizesmart.html" />
-    <EmbeddedResource Include="Web\autoorganizesmart.js" />
-    <EmbeddedResource Include="Web\autoorganizemovie.html" />
-    <EmbeddedResource Include="Web\autoorganizemovie.js" />
-    <EmbeddedResource Include="Web\autoorganizetv.html" />
-    <EmbeddedResource Include="Web\autoorganizetv.js" />
-    <EmbeddedResource Include="Web\fileorganizer.js" />
-    <EmbeddedResource Include="Web\fileorganizer.template.html" />
+    <EmbeddedResource Include="Web\autoorganizelog.html"/>
+    <EmbeddedResource Include="Web\autoorganizelog.js"/>
+    <EmbeddedResource Include="Web\autoorganizesmart.html"/>
+    <EmbeddedResource Include="Web\autoorganizesmart.js"/>
+    <EmbeddedResource Include="Web\autoorganizemovie.html"/>
+    <EmbeddedResource Include="Web\autoorganizemovie.js"/>
+    <EmbeddedResource Include="Web\autoorganizetv.html"/>
+    <EmbeddedResource Include="Web\autoorganizetv.js"/>
+    <EmbeddedResource Include="Web\fileorganizer.js"/>
+    <EmbeddedResource Include="Web\fileorganizer.template.html"/>
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.8.0-alpha3"/>
-    <PackageReference Include="Jellyfin.Naming" Version="10.8.0-alpha3"/>
+    <PackageReference Include="Jellyfin.Controller" Version="10.*-*"/>
```

Comment:
Hello, I got some compile error such like "cannot resolve xxx property" when using 10.\*-\*, thats why I changed the version. Could you please tell me why it happened if you compile successful by using \*-\* pattern.

Reply:
The ide complains, but the build command works. Since we use the build command for CI and deployment, that’s what we keep it as

Reply:
Actually I built in command line because I haven't installed ide. I edited the code in vs code and built it in command line on win10. It seems failed by using the package of  10.7.* version and function called the new api added in 10.8.*. What did I miss?

Reply:
Never mind, I guess the api called added in early version, and remove in newest version since 10.8.* still on alpha.